### DISCUSSION TITLE
1440p modern resizable issue

### DISCUSSION THREAD

**Discussion by bearryj** [runelite/runelite#17179] [Q&A]
I play 1440p, and I want to scale the UI with 'Stretched Mode' to where on modern resizable the icons for inv, equip, prayer, etc. will stack on top of each other instead of spread out on a flat line on the bottom of the screen. This is possible to do correctly if you open the runelite side panel and set the scaling to 141%. The problem lies when you close the runelite side panel, you then have to set the scaling to 167% to get the icon to stack but at 167% the UI is massive and the inv overlaps into the minimap and the chat becomes very large. I need a way to lock UI at a certain scaling position. That way when I open and close the runelite side panel it doesn't shift my icons.

**Comment by Alexsuperfly** [comment]
interface styles has an always stack option

**Reply by bearryj** [reply]
thank you so much
